src/CreateEventTypeView/CreateEventTypeView.js

```diff
@@ -5,103 +5,185 @@ import {Container} from "react-grid-system";
 import RaisedButton from "material-ui/RaisedButton";
 import IconSave from "material-ui/svg-icons/content/save";
 import IconCancel from "material-ui/svg-icons/navigation/cancel";
-import { Row, Col } from 'react-grid-system';
+import {Row, Col} from 'react-grid-system';
 import Header from './../Header';
 import {css} from "aphrodite";
 import AppStyles from "./../AppStyles";
+import config from '../config/config.js';
+import ErrorMessage from '../Utils/ErrorMessage.js';
 
 
 class CreateEventTypeView extends ConnectionComponent {
-    constructor(props) {
-        super(props);
-        this.state = {
-            attributes : [{id: '0', value: ''}],
-            lastAttributeId: 0
-        };
-        this.nextAttributeId = 1;
-    }
+	constructor(props) {
+		super(props);
+		this.state = {
+			attributes: [{id: '0', value: ''}],
+			lastAttributeId: 0,
+			eventTypeName: '',
+			eventTypeNameErrorText: '',
+			eventTypeTimestampAttribute: '',
+			eventTypeTimestampAttributeErrorText: ''
+		};
+		this.nextAttributeId = 1;
+		this.handleChangeEventTypeName = this.handleChangeEventTypeName.bind(this);
+		this.handleChangeEventTypeTimestampAttribute = this.handleChangeEventTypeTimestampAttribute.bind(this);
+		this.submitForm = this.submitForm.bind(this);
+	}
 
-    onInputChange(event) {
-        const currentAttributeId = event.target.id.toString();
-        const updatedAttributes = this.state.attributes;
-        const attributeIds = updatedAttributes.map(attribute => {
-            return attribute.id;
-        });
-        const currentAttributeIndex = attributeIds.indexOf(currentAttributeId);
+	handleChangeEventTypeName(event) {
+		if(event.target.value) {
```

Comment:
Prevents the deletion of last character:

1. Type something in input field
1. Try to delete everything by pressing `back` multiple times, or marking all and then `back`
1. You will see that at least the last character is quite rebellious

This is slightly inconvenient for the user.